dfc/tests/regression_test.go

```
@@ -813,7 +650,7 @@ func regressionRebalance(t *testing.T) {
 	for sid = range smap.Tmap {
 		break
 	}
-	unregisterTarget(sid, t)
+	client.UnregisterTarget(proxyurl, sid)
```

returned error is not checked